onelogin_aws_cli/__init__.py

```diff
@@ -6,39 +6,13 @@
 import configparser
 import getpass
 import os
-import xml.etree.ElementTree as ET
+import xml.etree.ElementTree as ElementTree
 
 import boto3
 from onelogin.api.client import OneLoginClient
 
 CONFIG_FILENAME = ".onelogin-aws.config"
-
-
-def user_choice(question, options):
-    """
-    Prompt a user with a question and a specific set of possible responses
-    :param question: Specifying context for the user to select an option
-    :param options: A list of options for the user to select from
-    :return:
-    """
-
-    print(question + "\n")
-    option_list = ""
-    for i, option in enumerate(options):
-        option_list += ("{}. {}\n".format(i + 1, option))
-    selection = None
-    while selection is None:
-        print(option_list)
-        choice = input("? ")
-        try:
-            val = int(choice) - 1
-            if val in range(0, len(options)):
-                selection = options[val]
-            else:
-                print("Invalid option")
-        except ValueError:
-            print("Invalid option")
-    return selection
+DEFAULT_CONFIG_PATH = os.path.expanduser("~/{}".format(CONFIG_FILENAME))
```

Comment:
I may be getting ahead of myself but this opening the door to a hierarchy of paths (including environment variables) to look for configs. This could allow for system-wide configs installed by admins, so the user config file only contains user-specific configurations.